frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelineDeleteModal.ts

```diff
@@ -0,0 +1,13 @@
+import { DeleteModal } from '~/__tests__/cypress/cypress/pages/components/DeleteModal';
+
+class PipelineDeleteModal extends DeleteModal {
+  constructor() {
+    super('Delete pipelines?');
```

Comment:
Looks like this page object is used for both the single delete and the bulk delete, correct? I think both of these modals have different titles, so I'm not sure if this constructor makes sense for both.